CHANGELOG.md

```diff
@@ -4,6 +4,62 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.46.0
+
+- **NEW** feat(sveltekit): Add Performance Monitoring for SvelteKit
```

Comment:
What about instead of adding `**NEW**` here, we bolden the first part, so e.g. `**feat(sveltekit)**: Add ...`?